test/prototype_transforms_kernel_infos.py

```diff
@@ -62,7 +67,7 @@ def wrapper(image_tensor, *other_args, **kwargs):
 
 
 def sample_inputs_horizontal_flip_image_tensor():
-    for image_loader in make_image_loaders(dtypes=[torch.float32]):
+    for image_loader in make_image_loaders(sizes=[None], dtypes=[torch.float32]):
```

Comment:
Looks strange `sizes=[None]`, what does that do ?

Reply:
`sizes` is a sequence of image sizes, i.e. height and width of the images to be created. By default we use

https://github.com/pytorch/vision/blob/c0911e31d367eca9600541bd634564c73753abbf/test/prototype_common_utils.py#L165-L168

where `None` means "I don't care, just something small"

https://github.com/pytorch/vision/blob/c0911e31d367eca9600541bd634564c73753abbf/test/prototype_common_utils.py#L172-L173

Together, `sizes=[None]` means, we don't want to test the full matrix on square, landscape, and portrait images here, but only on random image_sizes.